adal/build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec

```diff
@@ -32,15 +32,17 @@ ADAL.NET takes advantage of Windows Server Active Directory and Azure Active Dir
         <dependency id="System.Runtime.Serialization.Json" version="4.3.0" exclude="Build,Analyzers" />
         <dependency id="System.Runtime.Serialization.Primitives" version="4.3.0" exclude="Build,Analyzers" />
         <dependency id="System.Security.Cryptography.X509Certificates" version="4.3.0" exclude="Build,Analyzers" />
-        <dependency id="System.Xml.XDocument" version="4.3.0" exclude="Build,Analyzers" />
+        <dependency id="System.Xml.XDocument" version="4.3.0" exclude="Build,Analyzers" />        
       </group>
       <group targetFramework="MonoAndroid8.1">
+        <dependency id="Xamarin.Android.Support.CustomTabs" version="27.0.2" exclude="Build,Analyzers" />
+        <dependency id="Xamarin.Android.Support.v7.AppCompat" version="27.0.2" exclude="Build,Analyzers" />
       </group>
       <group targetFramework="UAP10.0">
         <dependency id="Microsoft.NETCore.UniversalWindowsPlatform" version="6.0.8" exclude="Build,Analyzers" />
-        <dependency id="System.Runtime.Serialization.Json" version="4.0.3" exclude="Build,Analyzers" />
-        <dependency id="System.Runtime.Serialization.Primitives" version="4.1.1" exclude="Build,Analyzers" />
-        <dependency id="System.Xml.XDocument" version="4.0.11" exclude="Build,Analyzers" />
+        <dependency id="System.Runtime.Serialization.Json" version="4.3.0" exclude="Build,Analyzers" />
```

Comment:
Somewhat unrelated fix, as I was getting NuGet warnings about these referneces. Updated UAP references to 4.3.0

Reply:
great! thanks.